### PR TITLE
feat: support multiple streams for item menu

### PR DESCRIPTION
**Pull request by noaione**
The following context menu has been adjusted for multiple media streams:
- Delete (should correctly delete the selected stream)
- Media Info (hide the stream selector if media index is provided)
- Metadata editor (should correctly show/edit the selected stream)

The multi-supported context menu would only show on the following:
- Item page
- Playback bar (audio mode)

Anything outside just default to the first media source.

### CODE REVIEW

File: frontend/src/components/Item/Metadata/MetadataEditor.vue [hunk 1]
```diff
@@ -36,6 +36,14 @@
         v-model="tabName"
         class="pa-2 flex-fill">
         <VWindowItem value="general">
+          <VSelect
+            v-if="contentOptions.length > 1"
```
Comment (ferferga) [inline frontend/src/components/Item/Metadata/MetadataEditor.vue:40]: Not sure if this is what you wanted? I guess we want here a disabled selector anyway?
```suggestion
            v-if="contentOptions.length > 0"
```

Can you also explain to me what `contentOptions` are supposed to be? This API still baffles me to this day 😅

File: frontend/src/components/Item/Metadata/MetadataEditorDialog.vue [hunk 2]
```diff
@@ -14,7 +14,7 @@
 <script setup lang="ts">
 import { ref } from 'vue';
 
-defineProps<{ itemId: string }>();
+defineProps<{ itemId: string; }>();
```
Comment (ferferga) [inline frontend/src/components/Item/Metadata/MetadataEditorDialog.vue:17]:
```suggestion
defineProps<{ itemId: string }>();
```

File: frontend/src/components/Item/Metadata/MetadataEditor.vue [hunk 3]
```diff
@@ -221,7 +229,12 @@ import { getItemUpdateApi } from '@jellyfin/sdk/lib/utils/api/item-update-api';
 import { format, formatISO } from 'date-fns';
 import { useDateFns, useRemote, useSnackbar } from '@/composables';
 
-const props = defineProps<{ itemId: string }>();
+type ContentOption = {
+  value: string;
+  key: string;
+};
+
+const props = defineProps<{ itemId: string; }>();
```
Comment (ferferga) [inline frontend/src/components/Item/Metadata/MetadataEditor.vue:237]:
```suggestion
const props = defineProps<{ itemId: string }>();
```

File: frontend/src/components/Item/Identify/IdentifyDialog.vue [hunk 4]
```diff
@@ -105,7 +105,7 @@ interface IdentifyField {
   value?: string | null;
 }
 
-const props = defineProps<{ item: BaseItemDto; mediaSourceIndex?: number }>();
+const props = defineProps<{ item: BaseItemDto; }>();
```
Comment (ferferga) [inline frontend/src/components/Item/Identify/IdentifyDialog.vue:108]:
```suggestion
const props = defineProps<{ item: BaseItemDto }>();
```

File: frontend/src/components/Item/Metadata/MetadataEditor.vue (hunk 1 above)
Comment (noaione) [reply frontend/src/components/Item/Metadata/MetadataEditor.vue:40]: I actually do not know too lol. There is a lot of weird thing in the API honestly.

**Pull request merged**